### PR TITLE
Update disruptions.md

### PR DESCRIPTION
**Pull request by jiafujiang**
Fix a small mistake.

### CODE REVIEW

**Comment by jiafujiang** [comment]
I have already signed a CLA.

发件人: Kubernetes Prow Robot<mailto:notifications@github.com>
发送时间: 2020年3月29日 21:10
收件人: kubernetes/website<mailto:website@noreply.github.com>
抄送: Jiafu Jiang<mailto:jiafujiang@outlook.com>; Mention<mailto:mention@noreply.github.com>
主题: Re: [kubernetes/website] Update disruptions.md (#19939)


Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

📝 Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.

It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.


  *   If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address. Check your existing CLA data and verify that your email is set on your git commits<https://help.github.com/articles/setting-your-email-in-git/>.
  *   If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
  *   If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: https://support.linuxfoundation.org/
  *   Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: login-issues@jira.linuxfoundation.org<mailto:login-issues@jira.linuxfoundation.org>

Instructions for interacting with me using PR comments are available here<https://git.k8s.io/community/contributors/guide/pull-requests.md>. If you have questions or suggestions related to my behavior, please file an issue against the kubernetes/test-infra<https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:> repository. I understand the commands that are listed here<https://go.k8s.io/bot-commands>.

**Comment by jiafujiang** [comment]
I signed it

**Comment by jiafujiang** [comment]
I signed it.

**Comment by jiafujiang** [comment]
I signed it.

File: content/zh/docs/concepts/workloads/pods/disruptions.md [hunk 1]
```diff
@@ -505,7 +505,7 @@ the nodes in your cluster, such as a node or system software upgrade, here are s
 - 故障转移到另一个完整的副本集群。
    -  没有停机时间，但是对于重复的节点和人工协调成本可能是昂贵的。
 - 编写可容忍干扰的应用程序和使用 PDB。
-   - 没有停机
+   - 没有停机时
```
Comment (xichengliudui) [inline content/zh/docs/concepts/workloads/pods/disruptions.md]: 结合上下文，我认为翻译为 "不停机" 更合适。
/lgtm

**Pull request merged**